.gitlab/pkg_metrics.yml

```diff
@@ -226,11 +239,15 @@ check_pkg_size-a7:
     - iot_agent_suse-x64
   variables:
     MAJOR_VERSION: 7
-    FLAVORS: "datadog-agent datadog-iot-agent datadog-dogstatsd"
+    FLAVORS: "datadog-agent datadog-iot-agent datadog-dogstatsd datadog-heroku-agent"
   before_script:
+    # ["datadog-heroku-agent"]="20000000" should probably replaced by "15000000" (or
+    # less if we remove system-probe as we should). "20000000" is needed on the very
+    # first commit until we have a public stable build.
```

Comment:
```suggestion
    # FIXME: ["datadog-heroku-agent"]="20000000" should probably replaced by "15000000"
    # "20000000" is needed as of now because of a large bump in size in system-probe in 7.35
    # If this happens often we could exclude system-probe from the heroku build, it's not used.
```